src/terminal/adapter/adaptDispatch.cpp

```diff
@@ -3537,7 +3469,7 @@ bool AdaptDispatch::AssignColor(const DispatchTypes::ColorItem item, const VTInt
         _renderSettings.SetColorAliasIndex(ColorAlias::FrameBackground, bgIndex);
         break;
     default:
-        return false;
+        break;
```

Comment:
This is another case where I think we should `return` rather than `break`, otherwise we're triggering an unnecessary redraw below.